docs/xss.rst

```diff
@@ -0,0 +1,95 @@
+Performing XSS Vulnerability Audits
+==================================
+
+The bok-choy framework includes the ability to perform XSS (cross-site scripting) audits on
+web pages using a short XSS locator defined in
+https://www.owasp.org/index.php/XSS_Filter_Evasion_Cheat_Sheet#XSS_Locator.
+
+Leverage Your Existing Tests and Fail on XSS Vulnerability Errors
+-------------------------------------------------------------
+
```

Comment:
Suggestions: 
1. Modify the heading in line 8 to something like "Trigger XSS Vulnerability Audits in Existing Tests".
2. Move your first sentence from line 19 to create an intro after the heading. "You might already have some bok-choy tests written for your web application. To leverage existing bok-choy tests and have them fail on finding XSS vulnerabilities, follow these steps." 
3. Add a set of numbered steps after your intro sentence.
4. Move the prereq in your note(line 16-17) into step 1. 

Steps would look something like this:
1. "Inject the XSS_INJECTION string defined in... into your page content." 
2. "Set the VERIFY_XSS environment variable ..."
    (Move code example here: ```export VERIFY_XSS = True```)

    An XSS audit is triggered whenever a page object's ```q``` method is called.
    If errors are found on the page, an XSS ExposureError is raised.